app/models/media_object.rb

```diff
@@ -179,14 +230,14 @@ def report_missing_attributes
   end
 
   def set_media_types!
-    mime_types = master_file_solr_docs.reject { |mf| mf["file_location_ssi"].blank? }.collect do |mf|
+    mime_types = section_solr_docs.reject { |mf| mf["file_location_ssi"].blank? }.collect do |mf|
```

Comment:
For consistency, should the `mf`s here and in the "set_resource_types!" method below all be changed to `section`?